src/styles/main.scss

```diff
@@ -0,0 +1,105 @@
+@import "stars";
+
+$color--text: #060b35;
+$color--text--title: #616070;
+$width--card--image: 160px;
+$height--card--image: 134px;
+$width--product-cards: 200px;
+$border--thickness: 1px;
+$background--color--card: #fff;
+$height--button: 40px;
+$width--button: 166px;
+$color--button: #00acdc;
```

Comment:
I see you're familiar with imports. So you can improve your code more. 
Move variables to a separated file `variables.scss`
Move card block styles to a separated file `card.scss`
Import these files into main.scss